index.js

```diff
@@ -31,10 +52,11 @@ module.exports = {
         }
       };
 
-      return new TemplateLinter(tree, {
+      return TemplateLinter.create(tree, {
         annotation: 'TemplateLinter',
         templatercPath: this.project.root + '/.template-lintrc',
-        generateTestFile: this.project.generateTestFile,
+        testGenerator: this._options.testGenerator,
+        group: (this._options.group !== false) ? type : undefined,
```

Comment:
I don't find it valuable to have two concepts for the same word here.  The broccoli plugin **only exists** to support the addon....